hack/jenkins/upload-to-gcs.sh

```diff
@@ -70,13 +70,13 @@ if [[ -n "${remote_git_repo}" ]]; then
 fi
 
 if [[ ${JOB_NAME} =~ -pull- ]]; then
-  : ${JENKINS_GCS_LOGS_PATH:="gs://kubernetes-jenkins/pr-logs/pull/${GCS_SUBDIR}${ghprbPullId:-unknown}"}
-  : ${JENKINS_GCS_LATEST_PATH:="gs://kubernetes-jenkins/pr-logs/directory"}
-  : ${JENKINS_GCS_LOGS_INDIRECT:="gs://kubernetes-jenkins/pr-logs/directory/${JOB_NAME}"}
+  : "${JENKINS_GCS_LOGS_PATH:="gs://kubernetes-jenkins/pr-logs/pull/${GCS_SUBDIR}${ghprbPullId:-unknown}"}"
+  : "${JENKINS_GCS_LATEST_PATH:="gs://kubernetes-jenkins/pr-logs/directory"}"
+  : "${JENKINS_GCS_LOGS_INDIRECT:="gs://kubernetes-jenkins/pr-logs/directory/${JOB_NAME}"}"
```

Comment:
Why did it change and what did it fix:
```
In ./hack/jenkins/upload-to-gcs.sh line 73:
  : ${JENKINS_GCS_LOGS_PATH:="gs://kubernetes-jenkins/pr-logs/pull/${GCS_SUBDIR}${ghprbPullId:-unknown}"}
    ^-- SC2223: This default assignment may cause DoS due to globbing. Quote it.

For more information:
  https://www.shellcheck.net/wiki/SC2223
```

Example:
**Problematic code**:
```
: ${COLUMNS:=80}
```
**Correct code**:
```
: "${COLUMNS:=80}"
```